src/workerd/api/node/tests/sidecar-supervisor.mjs

```diff
@@ -51,12 +50,6 @@ function waitForListening(port) {
         resolve();
       })
       .once('error', (err) => console.log('waiting for sidecar...', err.code));
-
-    if (attempts > CONNECT_POLL_MAX_ATTEMPTS) {
-      reject(
-        `Sidecar failed to listen to port ${port} after ${attempts} attempts.`
```

Comment:
You don't need attempts now I guess?

Reply:
True

Reply:
Ah no I see what you mean, I'm getting rid of the local variable now.